python/across_api/auth/api.py

```diff
@@ -79,14 +84,38 @@ async def __call__(self, request: Request) -> None:
             raise AssertionError("No credentials passed.")
 
 
-security = JWTBearer(
+JWTBearerSecurity = JWTBearer(
     scheme_name="ACROSS API Authorization",
     description="Enter your access token.",
 )
-JWTBearerDep = [Depends(security)]
 
 
-@app.get("/auth/verify", dependencies=JWTBearerDep)
+async def ScopeAuthorize(
+    security_scopes: SecurityScopes,
+    access_token: Annotated[dict, Depends(JWTBearerSecurity)],
+):
```

Comment:
Why are all of these functions CamelCase rather than snake_case? That's usually reserved for classes.

Reply:
I agree, this has been fixed to follow the snake_case for variable names.